src/Bridge/Doctrine/DBAL/Types/AbstractEnumType.php

```diff
@@ -47,6 +52,8 @@ protected function onNullFromPhp()
      * {@inheritdoc}
      *
      * @param EnumInterface $value
+     * @psalm-param EnumInterface<T> $value
```

Comment:
The value can actually be `null` here too.

Reply:
Good catch, thanks!